### ISSUE TITLE
UnhandledPromiseRejectionWarning when opening any template.json in vscode

### ISSUE THREAD

**Issue by hjst** [threadheap/serverless-ide-vscode#7]
Just installed 0.2.1 and I'm getting this error in the vscode output tab whenever I open a SAM template file:

```
[Error - 6:23:41 PM] (node:19560) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'schema' of null
    at applyGlobalsConfigMutations (/Users/henry/.vscode/extensions/threadheap.serverless-ide-vscode-0.2.1/node_modules/@serverless-ide/language-server/out/language-service/services/jsonSchema/mutation.js:9:31)
    at Object.exports.applyDocumentMutations (/Users/henry/.vscode/extensions/threadheap.serverless-ide-vscode-0.2.1/node_modules/@serverless-ide/language-server/out/language-service/services/jsonSchema/mutation.js:41:12)
    at JSONSchemaService.<anonymous> (/Users/henry/.vscode/extensions/threadheap.serverless-ide-vscode-0.2.1/node_modules/@serverless-ide/language-server/out/language-service/services/jsonSchema/index.js:411:31)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/henry/.vscode/extensions/threadheap.serverless-ide-vscode-0.2.1/node_modules/@serverless-ide/language-server/out/language-service/services/jsonSchema/index.js:4:58)
[Error - 6:23:41 PM] (node:19560) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
[Error - 6:26:56 PM] (node:19560) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'schema' of null
    at applyGlobalsConfigMutations (/Users/henry/.vscode/extensions/threadheap.serverless-ide-vscode-0.2.1/node_modules/@serverless-ide/language-server/out/language-service/services/jsonSchema/mutation.js:9:31)
    at Object.exports.applyDocumentMutations (/Users/henry/.vscode/extensions/threadheap.serverless-ide-vscode-0.2.1/node_modules/@serverless-ide/language-server/out/language-service/services/jsonSchema/mutation.js:41:12)
    at JSONSchemaService.<anonymous> (/Users/henry/.vscode/extensions/threadheap.serverless-ide-vscode-0.2.1/node_modules/@serverless-ide/language-server/out/language-service/services/jsonSchema/index.js:411:31)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/henry/.vscode/extensions/threadheap.serverless-ide-vscode-0.2.1/node_modules/@serverless-ide/language-server/out/language-service/services/jsonSchema/index.js:4:58)
[Error - 6:26:56 PM] (node:19560) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
```

Guessing there's something borked with my install but I'm not familiar enough with code's js/ts to know where to begin :-(

**Comment by pavelvlasov** [comment]
Thank for raising @hjst! I assume you're trying to open `template.json` file. Unfortunately, there is no support of json templates.

**Comment by vco1** [comment]
I can reproduce, and actually have this issue since installing Serverless IDE 0.2.1, with a standard `template.yaml` file.

**Comment by pavelvlasov** [comment]
@vco1, I probably have a clue, but would be great if you can share your template file name and serverless ide config

**Comment by vco1** [comment]
@pavelvlasov, I'm using a template (a.o.) that's the result of the following command `sam init --runtime python3.7 --name test-sam-app`
You comment about config triggered further investigation. I wasn't aware there were settings for Serverless IDE. So everything was still as default. It turns out the default SAM template file extension for Serverless IDE is `.sam.yaml` while the sam cli generates `template.yaml`. When I change the filename to template.sam.yaml the errors are gone.

**Comment by pavelvlasov** [comment]
@vco1 the fix is out with 0.2.3 release. Please ping me, if it happens again
